.github/workflows/CI-main.yml

```diff
@@ -111,7 +113,7 @@ jobs:
   build-internal:
     name: Build Internal App
     needs: resolve-values
-    uses: bitwarden/ios/.github/workflows/build.yml@main
+    uses:  bitwarden/ios/.github/workflows/build.yml@main
```

Comment:
@Eeebru is linter not catching these?

Reply:
The linter haven't implemented that, I might look into that @vvolkgang